redisinsight/ui/src/pages/rdi/instance-list/RdiInstancesListWrapper.spec.tsx

```diff
@@ -117,6 +117,14 @@ describe('RdiInstancesListWrapper', () => {
             ...state.rdi.instances,
             data: mockInstances
           }
+        },
+        app: {
+          ...state.app,
+          context: {
+            ...state.app.context,
+            lastPage: '',
+            contextRdiInstanceId: '',
+          }
```

Comment:
please, mock only exact selector, not useSelector

Reply:
Got it.